bintrace.c

```diff
@@ -393,8 +451,41 @@ int main(int argc, char **argv)
 	struct process_t tracee;
 	struct user_regs_struct regs;
 	uint64_t ins;
-
-	if (argc < 3) {
+	
+	int opt;
+	struct flags cmd_flags;
```

Comment:
Let's make them a global variable. We will probably use then often to alter behavior of the functions. This state is global for the whole program anyway, no need to pass them every time we call a function. I believe this will make function args list shorter, thus easier to read.